src/libraries/Microsoft.PowerFx.Core/Parser/ParseFormulasResult.cs

```diff
@@ -139,5 +139,13 @@ public UDFArg(IdentToken nameIdent, IdentToken typeIdent, Token colonToken, int
             ArgIndex = argIndex;
             ColonToken = colonToken;
         } 
+    }
+
+    internal enum UserDefinitionType
```

Comment:
>UserDefinitionType [](http://example.com/codeflow?start=18&length=18)

Is this a flags enum (i.e., can variables of this type be initialized as something like UserDefinitionType.UDF | UserDefinitionType.NamedFormula)? If so, then please add the `[Flags]` attribute; if not, then there's no reason to initialize the values with 0/1/2/4 - we can use the defaults from .NET.